packages/cli/plopfile.js

```diff
@@ -0,0 +1,46 @@
+const promptDirectory = require('inquirer-directory');
+
+module.exports = function(plop) {
+  plop.setPrompt('directory', promptDirectory);
+
+  plop.setGenerator('ngxs', {
+    description: 'Create new store',
+    prompts: [
+      {
+        type: 'input',
+        name: 'name',
+        message: 'Write store name, please...'
+      },
+      {
+        type: 'list',
+        name: 'storeType',
+        choices: ['Empty Store', 'Filled Store'],
+        message: 'Which store do you need? 😊'
+      },
+      {
+        type: 'directory',
+        name: 'directory',
+        message: 'Choose a directory..',
```

Comment:
How about just `Directory:`. Again the `...` makes me think I'm waiting on something. Also would be good to say `(Default CWD)`

Reply:
done